platform/security/platform-security-core-api/src/main/java/ddf/security/samlp/SimpleSign.java

```diff
@@ -68,6 +72,20 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SimpleSign.class);
 
+  // Should be removed in favor of a lookup through third-party library when one is found.
+  private static final Map<String, String> RFC5698_DSSC_OID_TO_URI_MAP =
```

Comment:
This change, and the rest in `SimpleSign`, are fallout from the library upgrades. They do improve DDF, by deferring to the _correct_ signature algorithm in more cases instead of falling back to SHA1withRSA/SHA1withDSA all the time.